src/concurrency/vector_clock.rs

```diff
@@ -42,7 +47,37 @@ const SMALL_VECTOR: usize = 4;
 
 /// The type of the time-stamps recorded in the data-race detector
 /// set to a type of unsigned integer
```

Comment:
that comment is odd. Is it just explaining that this is a `u32`?

Reply:
This used to just be a type alias, I just failed to update the docs.